src/octoprint/printer/standard.py

```diff
@@ -650,6 +653,13 @@ def get_sd_files(self, *args, **kwargs):
 			return []
 		return map(lambda x: (x[0][1:], x[1]), self._comm.getSdFiles())
 
+	def success_hook_sdcopy(self, payload):
```

Comment:
my goal, with this change, is to be able to more discreetly replace the way to upload to the SD card, reminders "if successful" and "in case of error" allow me to follow the normal process if successful or of error (notification ..)
in fact I could possibly change the return of the hook by a Boolean.
Initially, I planned to return the name of the file, but it seems that changing the name of the file is problematic.
so I'm going to change the signature of the method to return a boolean

Reply:
i remove the callbacks